Detectors/ITSMFT/MFT/tracking/src/Tracker.cxx

```diff
@@ -288,7 +292,7 @@ void Tracker<T>::findTracksLTF(ROframe<T>& event)
       clsInLayer1 = it1 - event.getClustersInLayer(layer1).begin();
 
       // loop over the bins in the search window
-      for (const auto& binS : getBinsS()[layer1][layer2 - 1][cluster1.indexTableBin]) {
+      for (const auto& binS : mBinsS.get()[0][layer1][layer2 - 1][cluster1.indexTableBin]) {
```

Comment:
same here and below, please change `mBinsS.get()[0]` to `(*mBinsS,get())`

Reply:
yes